.github/workflows/build-and-test-aarch64-darwin.yml

```diff
@@ -0,0 +1,41 @@
+name: build and test on aarch64
+on: [push, pull_request]
+jobs:
+  test-on-macos-aarch64:
+    runs-on: macos-14
+    name: test on macos-14-aarch64
+    steps:
+      - name: install prerequisites
+        run: |
+          rustup target add armv7-unknown-linux-gnueabihf
+          brew tap messense/macos-cross-toolchains
+          brew install \
+            armv7-unknown-linux-gnueabihf \
```

Comment:
Why do we need `armv7-unknown-linux-gnueabihf` on aarch64 macOS?  Is this a typo?

Reply:
It is a leftover from me trying to run the armv7 in docker. I ended up keeping the QEMU armv7 workflow instead. Removed.